src/core.c/CompUnit/Repository/Staging.pm6

```diff
@@ -0,0 +1,73 @@
+class CompUnit::Repository::Staging is CompUnit::Repository::Installation {
+    has Str $.name;
+    has CompUnit::Repository $!parent;
+
+    submethod TWEAK (--> Nil) {
+       $!parent = self.next-repo;
```

Comment:
Can not use `CURR.repository-for-name($!name)` here because when including `Staging` (e.g. `raku -I 'staging#name(site)#/tmp/stage'`) `$*REPO` wont get initialized and `CURR.setup-repositories` stuck in a loop